recipe/bld.bat

```diff
@@ -30,7 +30,7 @@ echo y | configure.exe -prefix %LIBRARY_PREFIX% ^
                        -L %LIBRARY_LIB% ^
                        -I %LIBRARY_INC% ^
                        -system-libjpeg ^
-                       -platform win32-msvc%VS_YEAR% ^
+                       -platform win32-msvc%VS_YEAR%
 
 nmake
```

Comment:
Don't we need to run `bin\qmake -r QT_BUILD_PARTS="libs tools"` before `nmake`?

Reply:
We do, probably I deleted that by accident